### ISSUE TITLE
Store Tesla credentials on network share instead of Pi

### ISSUE THREAD

**Issue by BrianGilbert** [marcone/teslausb#460]
as per subject

**Comment by vectorvirge** [comment]
Yeah - anyway we can point it to the network share to pull this? I'd rather store it on my home network versus the RPi

**Comment by MarcFinns** [comment]
+1

**Comment by Arby69** [comment]
No acknoledge so far?
+1

**Comment by marcone** [comment]
I think it's a good idea but it's not high on my priority list. Anybody want to contribute a PR?

**Comment by nelsongraca** [comment]
@marcone I may contribute as I like the idea of not having credentials on the car.

**Comment by nelsongraca** [comment]
To those interested, what kind of network share are we talking about? CIFS/SMB? SFTP? Something supported by RClone?

**Comment by MarcFinns** [comment]
I use CIFS/SMB

**Comment by Fredyy90** [comment]
This sounds like a very interesting feature, having a TXT file on my share, is defiantly nicer, than saving the data on the stick itself.
Since the credentials are only needed, when near my network, this should theoretically introduce no new issues.
